WalletWasabi/WabiSabi/Client/WabiSabiApiClient.cs

```diff
@@ -0,0 +1,63 @@
+using NBitcoin;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using WalletWasabi.Crypto;
+using WalletWasabi.WabiSabi.Backend.PostRequests;
+using WalletWasabi.WabiSabi.Crypto;
+using WalletWasabi.WabiSabi.Models;
+
+namespace WalletWasabi.WabiSabi.Client
+{
+	public class WabiSabiApiClient
+	{
+		public WabiSabiApiClient(WabiSabiClient wabiSabiClientAmount, WabiSabiClient wabiSabiClientWeight, IRequestHandler requestHandler)
+		{
+			WabiSabiClientAmount = wabiSabiClientAmount;
+			WabiSabiClientWeight = wabiSabiClientWeight;
+			RequestHandler = requestHandler;
+		}
+
+		public WabiSabiClient WabiSabiClientAmount { get; }
+		public WabiSabiClient WabiSabiClientWeight { get; }
```

Comment:
this is pure bikeshedding but maybe we should consider renaming things, having `WabiSabiClient` for low level credential operations and `WabiSabiApiClient` for high level coinjoin stuff seems really confusing

Reply:
I agree, it is really confusing. I will change it and I am open to suggestions.

Reply:
maybe `WabiSabiCredentialClient` and `WabiSabiCoinJoinClient` or something like that?

Reply:
I called it `CoordinatorClient` because I found it more clear. We can rename it again but I don't want to continue calling the things `WabiSabiXXX` because everything seems the same.

Reply:
If this client stores the information for a specific round, it can be CoinJoinRoundClient?

Reply:
`ArenaClient`